sdk/storage/azure-storage-blobs/src/blob_lease_client.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "azure/storage/blobs/blob_lease_client.hpp"
 
-#include <azure/core/azure_assert.hpp>
+#include "azure/core/internal/assert.hpp"
```

Comment:
```suggestion
#include <azure/core/internal/assert.hpp>
```

Reply:
We use `""` for headers within the package, and `<>` for headers out of the package.